### PR TITLE
add supergaelicdodgeball

### PR DESCRIPTION
**Pull request by zehmaluco**
# New Port for { Super Gaelic Dodgeball }

## Game Information
- **Title**: { Super Gaelic Dodgeball }
- **URL**: { https://shoehead.itch.io/sgdb }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by JeodC** [comment]
`port.json`, `readme.md`, and `cover.png`, `screenshot.png` all need to be in the same folder as the `.sh` file. Not inside the portfolder.

**Comment by zehmaluco** [comment]
> `port.json`, `readme.md`, and `cover.png`, `screenshot.png` all need to be in the same folder as the `.sh` file. Not inside the portfolder.

Done. :)

File: ports/supergaelicdodgeball/Super Gaelic Dodgeball.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then
```
Comment (Cebion) [inline ports/supergaelicdodgeball/Super Gaelic Dodgeball.sh:36]: gmloader does not use libgl

File: ports/supergaelicdodgeball/cover.png
Comment (Cebion) [inline ports/supergaelicdodgeball/cover.png:1]: cover has to be 640x480

File: ports/supergaelicdodgeball/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Super Gaelic Dodgeball.sh</path>
+    <name>Super Gaelic Dogdeball</name>
+    <desc>Mildly violent Dodgeball game made for Awful Winter Jam 2018</desc>
+    <releasedate>20220122T000000</releasedate>
+    <developer>Shoehead</developer>
+    <publisher>Icth.io</publisher>
```
Comment (Cebion) [inline ports/supergaelicdodgeball/gameinfo.xml:9]: Typo, and is itch really the publisher or just the hoster?
Hoster is not publisher

File: ports/supergaelicdodgeball/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        ],
+        "desc": "Mildly violent Dodgeball game made for Awful Winter Jam 2018",
+        "desc_md": null,
+        "inst": "Get the game on Itch.io and place all files on the "gamedata" portfolder.",
+        "inst_md": null,
+        "genres": [
+            "sports"
+        ],
+        "image": {},
+        "rtr": true,
```
Comment (Cebion) [inline ports/supergaelicdodgeball/port.json:22]: If this game is rtr it needs license files. Else it's not ready to run.

File: ports/supergaelicdodgeball/port.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Super Gaelic Dodgeball",
+        "porter": [
+            "Zehmaluco"
+        ],
+        "desc": "Mildly violent Dodgeball game made for Awful Winter Jam 2018",
+        "desc_md": null,
+        "inst": "Get the game on Itch.io and place all files on the "gamedata" portfolder.",
```
Comment (Cebion) [inline ports/supergaelicdodgeball/port.json:16]: Where exactly?

File: ports/supergaelicdodgeball/screenshot.png
Comment (Cebion) [inline ports/supergaelicdodgeball/screenshot.png:1]: Screenshot has to be 640x480

File: ports/supergaelicdodgeball/supergaelicdodgeball/log.txt
Comment (Cebion) [inline ports/supergaelicdodgeball/supergaelicdodgeball/log.txt:1]: log.txt can be deleted

**Review by Cebion — CHANGES_REQUESTED** [review]
README.MD is also missing.

Check the packaging guide on our website

https://portmaster.games/packaging.html

**Comment by zehmaluco** [comment]
Made the changes. Thanks for the patience, dudes. I'm still getting used to Git. :)

**Comment by zehmaluco** [comment]
Made the changes. 👍

**Review by Cebion — APPROVED** [review]

**Comment by zehmaluco** [comment]
Sorry, Cebion. I've didn't know I could only do a pull at a time and add one more project file on this pull. My bad, sorry 😛

**Comment by Cebion** [comment]
> Sorry, Cebion. I've didn't know I could only do a pull at a time and add one more project file on this pull. My bad, sorry 😛

Yeah you have to work with branches.

**Review by Cebion — DISMISSED** [review]

**Comment by zehmaluco** [comment]
@Cebion @JeodC i've updated the cover.png to look similar to Jeod template. Sorry for the back and forth on this. 🙇

**Review by Cebion — DISMISSED** [review]

**Review by kloptops — CHANGES_REQUESTED** [review]
port.json is borked. 😄

**Comment by zehmaluco** [comment]
> port.json is borked. 😄

Fixed. :)

**Review by Cebion — COMMENTED** [review]
But why :D

**Review by Cebion — DISMISSED** [review]

File: ports/supergaelicdodgeball/port.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "items_opt": null,
+    "attr": {
+        "title": "Super Gaelic Dodgeball",
+        "porter": [
+            "Zehmaluco"
+        ],
+        "desc": "Mildly violent Dodgeball game made for Awful Winter Jam 2018",
+        "desc_md": null,
+        "inst": "Get the game on shoehead.itch.io/sgdb and place all game files on the gamedata inside supergaelicdodgeball folder",
+        "inst_md": null,
```
Comment (kloptops) [inline ports/supergaelicdodgeball/port.json:17]: Probably a good idea to add "inst_md" with an actual link to the game:
```suggestion
        "inst_md": "Get the game on the [shoehead itch.io page](https://shoehead.itch.io/sgdb), and place all game files in the ports/supergaelicdodgeball/gamedata/ directory.",
```

**Comment by zehmaluco** [comment]
Fixed the inst_md on the port.json @Cebion @kloptops

**Review by kloptops — APPROVED** [review]

File: ports/supergaelicdodgeball/port.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": 1,
```
Comment (kloptops) [inline ports/supergaelicdodgeball/port.json:2]: this should be version 3.
```suggestion
    "version": 3,
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]

**Review by JeodC — APPROVED** [review]

**Pull request merged**